pkg/kubelet/dockertools/manager.go

```diff
@@ -894,10 +895,47 @@ func (dm *DockerManager) RunInContainer(containerID string, cmd []string) ([]byt
 		RawTerminal:  false,
 	}
 	err = dm.client.StartExec(execObj.ID, startOpts)
+	if err != nil {
+		return nil, err
+	}
+	tick := time.Tick(2 * time.Second)
+	for {
```

Comment:
Hrm, we were already blocking (detach is set to 'false') so to be honest this loop is probably never actually executed.

I'd rather not have a timeout here.